python/fbprophet/plot.py

```diff
@@ -495,3 +501,152 @@ def plot_cross_validation_metric(
+        y=fcst['yhat'],
+        mode='lines',
+        line=dict(color=prediction_color, width=line_width),
+        fillcolor=error_color,
+        fill='tonexty' if uncertainty else 'none'
+    ))
+    # Add upper bound
+    if uncertainty:
+        data.append(go.Scatter(
+            x=fcst['ds'],
+            y=fcst['yhat_upper'],
+            mode='lines',
+            line=dict(width=0),
+            fillcolor=error_color,
+            fill='tonexty',
+            hoverinfo='skip'
+        ))
+    # Add caps
+    if 'cap' in fcst and plot_cap:
+        data.append(go.Scatter(
+            name='Cap',
+            x=fcst['ds'],
+            y=fcst['cap'],
+            mode='lines',
+            line=dict(color=cap_color, dash='dash', width=line_width),
+        ))
+    if m.logistic_floor and 'floor' in fcst and plot_cap:
+        data.append(go.Scatter(
+            name='Floor',
+            x=fcst['ds'],
+            y=fcst['floor'],
+            mode='lines',
+            line=dict(color=cap_color, dash='dash', width=line_width),
+        ))
+    # Add trend
+    if trend:
+        data.append(go.Scatter(
+            name='Trend',
+            x=fcst['ds'],
+            y=fcst['trend'],
+            mode='lines',
+            line=dict(color=trend_color, width=line_width),
+        ))
+    # Add changepoints
+    if changepoints:
+        signif_changepoints = m.changepoints[
+            np.abs(np.nanmean(m.params['delta'], axis=0)) >= changepoints_threshold
+        ]
+        data.append(go.Scatter(
+            x=signif_changepoints,
+            y=fcst.loc[fcst['ds'].isin(signif_changepoints), 'trend'],
+            marker=dict(size=50, symbol='line-ns-open', color=trend_color,
+                        line=dict(width=line_width)),
+            mode='markers',
+            hoverinfo='skip'
+        ))
+
+    layout = dict(
+        showlegend=False,
+        yaxis=dict(
+            title=ylabel
+        ),
+        xaxis=dict(
+            title=xlabel,
+            type='date',
+            rangeselector=dict(
+                buttons=list([
+                    dict(count=7,
+                         label='1w',
+                         step='day',
+                         stepmode='backward'),
+                    dict(count=1,
+                         label='1m',
+                         step='month',
+                         stepmode='backward'),
+                    dict(count=6,
+                         label='6m',
+                         step='month',
+                         stepmode='backward'),
+                    dict(count=1,
+                         label='1y',
+                         step='year',
+                         stepmode='backward'),
+                    dict(step='all')
+                ])
+            ),
+            rangeslider=dict(
+                visible=True
+            ),
+        ),
+    )
+    fig = go.Figure(data=data, layout=layout)
+    return plotly.offline.iplot(fig)
```

Comment:
A bit tricky what to return here. This is one option which requires plotly.offline.init_notebook_mode() to be run first in notebooks. 
Another option is maybe to do what is done in Plotly Express using a new [ExpressFigure](https://github.com/plotly/plotly_express/blob/68d7ff0bbb53275ab0e0edf014caf8997ecc9c0c/plotly_express/_core.py#L22), but don't want to require a package only to use that (internal?) class and I don't fully get how it works myself. 
A third option is to return the Figure, and let the user do the iplot command. This would let users use plotly online and save to file (not that I know anyone who uses those options much).